### DISCUSSION TITLE
q-select - How to resize the menu

### DISCUSSION THREAD

**Discussion by AdamCMCalc** [quasarframework/quasar#16096] [General - Components / Directives / etc]
Hi,

I'm currently struggling to resize the menu's max-height on a q-select which has upwards of 50 potential options available for selection. At present, it's in its default format and occupying near 500px of vertical height, which I would like to reduce to around 100px.
I've tried inspecting and targeting selectors, but nothing seems to be working unfortunately.
I'd appreciate any help and/or advice.

Thanks in advance,
Adam.

**Comment by fjolla-d** [comment]
Hi @AdamCMCalc , I have the same issue. Did you manage to find a solution?

**Comment by ivanjaros** [comment]
in the documentation you have the virtual scroll properties described and in style section you have the popup styling available.

**Comment by rajat6235** [comment]
Hi, what worked for me is this very simple solution
<q-select
popup-content-style="height: 300px;"
